webonary-cloud-api/lambda/postDictionary.ts

```diff
@@ -111,6 +122,29 @@ export async function handler(
     dbClient = await connectToDB();
     const db = dbClient.db(DB_NAME);
 
+    // fulltext index (case and diacritic insensitive by default)
+    await db.collection(DB_COLLECTION_ENTRIES).createIndex(
+      {
+        [PATH_TO_ENTRY_MAIN_HEADWORD_VALUE]: 'text',
+        [PATH_TO_ENTRY_DEFINITION_VALUE]: 'text',
+      },
+      { name: 'wordsFulltextIndex', default_language: 'none' },
+    );
+
+    // case and diacritic insensitive index for semantic domains
+    // Vietnamese seem to have the most diacritics
```

Comment:
The comment here has the misspelling, but also seems out of place. It is only necessary because the variable names used in locale and strength don't indicate the purpose.

Reply:
good point :-) changed to more descriptive names